collect_app/src/androidTest/java/org/odk/collect/android/feature/formmanagement/MatchExactlyTest.java

```diff
@@ -53,40 +52,14 @@ public void whenMatchExactlyEnabled_clickingFillBlankForm_andClickingRefresh_get
     }
 
     @Test
-    public void whenMatchExactlyEnabled_getsLatestFormsFromServer_automaticallyAndRepeatedly() throws Exception {
```

Comment:
I decided to revise how the tests are written to drive out the new behaviour rather than just adding more. Along wiht the changed/new unit tests everything was TDD'd out and I'm pretty sure we didn't lose anything.